File: Lists/ProbList101/P1888/src/Main.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int[] arr = new int[]{sc.nextInt(), sc.nextInt(), sc.nextInt()};
        Arrays.sort(arr);
        int a = arr[0];
        int c = arr[2];
        int p = gcd(a, c);
        System.out.println(a/p + "/" + c/p);
    }

    private static int gcd(int x, int y) {
        int m = Math.max(x, y);
        int n = Math.min(x, y);
        int p = m % n;
        if (p == 0) return n;
        else return gcd(n, p);
    }
}

File: Lists/ProbList101/P1888/src/main.cpp
#include<bits/stdc++.h>
using namespace std;

int gcd(int x, int y) {
    int m = max(x, y);
    int n = min(x, y);
    int p = m % n;
    if (p == 0) return n;
    else return gcd(n, p);
}

int main()
{
    int arr[4];
    cin >> arr[0] >> arr[1] >> arr[2];
    sort(arr, arr+3);
    int a = arr[0];
    int c = arr[2];
    int p = gcd(a, c);
    cout << a/p << "/" << c/p;

    return 0;
}